src/components/OrgPostCard/components/PostEditModal.tsx

```
@@ -0,0 +1,284 @@
+              className={styles.inputField}
+              data-testid="video-upload"
+            />
+
+            {postFormState.attachments
+              .filter((a) => a.mimeType.startsWith('video/'))
+              .map((attachment, index) => (
+                <div key={index} className={styles.previewOrgPostCard}>
+                  <video controls data-testid="video-preview">
+                    <source src={attachment.url} type={attachment.mimeType} />
+                    {t('videoNotSupported')}
+                  </video>
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)